politeiawww/cmd/pictl/proposal.go

```diff
@@ -42,6 +42,10 @@ func printProposalFiles(files []rcv1.File) error {
 	if err == nil {
 		printf("%v\n", piv1.FileNameProposalMetadata)
 		printf("  Name: %v\n", pm.Name)
+		printf("  Amount: %v\n", pm.Amount)
+		printf("  Start Date: %v\n", pm.StartDate)
```

Comment:
The colons of these log statements should also be aligned.

Current
```
proposalmetadata.json
  Name: A Proposal Name 36c63802e2a9a8fb
  Amount: 2000000
  Start Date: 1629349200
  End Date: 1637128800
  Domain: research
```

Aligned
```
Proposal editted
Token    : 6c8a92176d0e3e28
Version  : 2
State    : unvetted
Status   : unreviewed
Timestamp: 07/20/2021 10:33am
Username : admin
```